forest/sycamore/read_audio.py

```diff
@@ -4,6 +4,7 @@
 import os
 from typing import Dict
 
+import librosa  # type: ignore
```

Comment:
We add modules with missing type hints to https://github.com/onnela-lab/forest/blob/develop/mypy.ini
More details: https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports